### PR TITLE
Add fragmenter of websocket frames

### PR DESCRIPTION
**Pull request by MichalChomo**
I haven't found a way to fragment outbound websocket frames according to a configured limit, so I added a fragmenter that does this. It uses `maxWebSocketFrameSize` config property.

### CODE REVIEW

File: server/src/main/java/org/atmosphere/nettosphere/NettyWebSocket.java [hunk 1]
```diff
@@ -18,12 +18,7 @@
 import io.netty.buffer.Unpooled;
 import io.netty.channel.Channel;
 import io.netty.channel.ChannelFutureListener;
-import io.netty.handler.codec.http.websocketx.BinaryWebSocketFrame;
-import io.netty.handler.codec.http.websocketx.CloseWebSocketFrame;
-import io.netty.handler.codec.http.websocketx.PingWebSocketFrame;
-import io.netty.handler.codec.http.websocketx.PongWebSocketFrame;
-import io.netty.handler.codec.http.websocketx.TextWebSocketFrame;
-import org.atmosphere.cpr.ApplicationConfig;
+import io.netty.handler.codec.http.websocketx.*;
```
Comment (jfarcand) [inline server/src/main/java/org/atmosphere/nettosphere/NettyWebSocket.java]: No wildcard. Please name all classes

File: server/src/main/java/org/atmosphere/nettosphere/NettyWebSocketFragmenter.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package org.atmosphere.nettosphere;
```
Comment (jfarcand) [inline server/src/main/java/org/atmosphere/nettosphere/NettyWebSocketFragmenter.java]: Add Apache license header.

File: server/src/main/java/org/atmosphere/nettosphere/NettyWebSocketFragmenter.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import io.netty.channel.Channel;
+import io.netty.handler.codec.http.websocketx.BinaryWebSocketFrame;
+import io.netty.handler.codec.http.websocketx.ContinuationWebSocketFrame;
+import io.netty.handler.codec.http.websocketx.TextWebSocketFrame;
+import io.netty.handler.codec.http.websocketx.WebSocketFrame;
+
+public class NettyWebSocketFragmenter {
+
+  private final Channel channel;
+  private final int maxFrameSize;
```
Comment (jfarcand) [inline server/src/main/java/org/atmosphere/nettosphere/NettyWebSocketFragmenter.java]: Minor, but tabs seems wrong.

**Comment by MichalChomo** [comment]
@jfarcand Thanks for a quick feedback. I committed the requested changes.

**Comment by jfarcand** [comment]
`3.2.0` has been released. Thanks and bring more patches like that anytime!

**Pull request merged**